index.js

```diff
@@ -258,6 +271,24 @@ class ServerlessDynamodbLocal {
         }).filter((n) => n);
     }
 
+    _listenForTermination() {
```

Comment:
As I understood, you have used underscore before the name of the function to differentiate it a private? But when I look at the code, so far this practice hasn't been followed. So shall we keep it without the underscore for the moment to keep the consistency?